pregred-gui/src/app/pages/home/Home.js

```diff
@@ -1,9 +1,255 @@
+                  <li>Prevent underweight and stunting</li>
+                  <li>Prevent vertical transmission of HIV/STIs</li>
+                  <li>Lower the risk of some forms of childhood cancers </li>
+                  <li>Lower the risk of type 2 diabetes and cardiovascular
+                    disease later in life.</li>
+                </ol>
+              </Modal>
+            </div>
+{/* Form 2 */}
+            <div className="home-div-models">
+              <Button type="primary" shape="round" onClick={() => setVisible2(true)}>
+                What are the factors you should be aware of to have a healthy baby?
+              </Button>
+              <Modal
+                title="What are the factors you should be aware of to have a healthy baby?"
+                centered
+                visible={visible2}
+                onOk={() => setVisible2(false)}
+                onCancel={() => setVisible2(false)}
+                width={800}
+              >
+                <ol>
+                  <li>Age</li>
+                  <li>Subfertility</li>
+                  <li>Oral health</li>
+                  <li>Mental health</li>
+                  <li>Gravidity</li>
+                  <li>Iron</li>
+                  <li>Nutrition and Supplements</li>
+                  <li>Calcium</li>
+                  <li>Folate (Folic Acid)</li>
+                </ol>
+
+              </Modal>
+            </div>
+{/* Form 3 */}
+            <div className="home-div-models">
+              <Button type="primary" shape="round" onClick={() => setVisible3(true)}>
+                What are the challenges you may face while achieving the goals to have a baby?
+              </Button>
+              <Modal
+                title="What are the challenges you may face
+                 while achieving the goals to have a baby?"
+                centered
+                visible={visible3}
+                onOk={() => setVisible3(false)}
+                onCancel={() => setVisible3(false)}
+                width={800}
+              >
+                <ol>
+                  <li>Time</li>
+                  <li>Poverty</li>
+                  <li>Facilities</li>
+                  <li>Family conditions</li>
+                </ol>
+              </Modal>
+            </div>
+{/* Form 4 */}
+            <div className="home-div-models">
+              <Button type="primary" shape="round" onClick={() => setVisible4(true)}>
+                Do you need consultancy?
+              </Button>
+              <Modal
+                title="Do you need consultancy?"
+                centered
+                visible={visible4}
+                onOk={() => setVisible4(false)}
+                onCancel={() => setVisible4(false)}
+                width={800}
+              >
+                <ol>
+                  <li>
+                    <a
+                      href="https://www.webmd.com/baby/considering-pregnancy-see-your-doctor-first">
```

Comment:
any outsource links which you are using should be opened in new tab. this link is opening in the same tab, change it to open in new tab

Reply:
ok